app/app.py

```diff
@@ -251,27 +251,34 @@ def main() -> None:
                     full_response = ""
                     message_placeholder = st.empty()
 
-                    # Query vector store
-                    context = query_vector_store(
-                        chroma_client=chroma_client,
-                        query_text=prompt,
-                        collection_name=COLLECTION_NAMES[0],
-                        n_results=N_CLOSEST_MATCHES,
-                        embedding_function=embed_function,
-                    )
+                    response = {}
+
+                    for collection in COLLECTION_NAMES:
+                        # Query vector store
+                        context = query_vector_store(
+                            chroma_client=chroma_client,
+                            query_text=prompt,
+                            collection_name=collection,
+                            n_results=N_CLOSEST_MATCHES,
+                            embedding_function=embed_function,
+                        )
+
+                        # Create a dict of prompt and context
+                        message = {"prompt_query": prompt, "context": context}
+
+                        # Query LLM by passing query and context
+                        assistant_response = query_llm(prediction_endpoint, message)
+
+                        response[collection] = assistant_response
 
-                    # Create a dict of prompt and context
-                    message = {"prompt_query": prompt, "context": context}
+                    # Two trailing-whitespaces are required at the end of the first response in order to output the response in the format we want
+                    full_response = f'''
+                        Here's what the NHS and Mind each have to say:
 
-                    # Query LLM by passing query and context
-                    assistant_response = query_llm(prediction_endpoint, message)
+                        {collection_name_map[COLLECTION_NAMES[0]]}: {response[COLLECTION_NAMES[0]]}  
```

Comment:
Similarly, this line and following line will change based on above implementation.